File: peek_plugin_unified_search/plugin-module/_private/tuples/AddIntValueActionTuple.ts
 import {addTupleType, Tuple, TupleActionABC} from "@synerty/vortexjs";
import {unifiedSearchTuplePrefix} from "../PluginNames";

@addTupleType
export class AddIntValueActionTuple extends TupleActionABC {
    static readonly tupleName = unifiedSearchTuplePrefix + "AddIntValueActionTuple";

    excludedKwId: number;
    offset: number;

    constructor() {
        super(AddIntValueActionTuple.tupleName)
    }
}

File: peek_plugin_unified_search/_private/tuples/AddIntValueActionTuple.py
from vortex.Tuple import addTupleType, TupleField
from vortex.TupleAction import TupleActionABC

from peek_plugin_unified_search._private.PluginNames import unifiedSearchTuplePrefix


@addTupleType
class AddIntValueActionTuple(TupleActionABC):
    __tupleType__ = unifiedSearchTuplePrefix + "AddIntValueActionTuple"

    excludedKwId = TupleField()
    offset = TupleField()
